Seeder/blacklists/models.py

```diff
@@ -46,3 +48,14 @@ def collect_urls_by_type(cls, blacklist_type):
         # blacklist urls is now list of contents
         urls_parsed = map(str.splitlines, blacklist_urls)
         return reduce(operator.add, urls_parsed, [])
+
+    @classmethod
+    def last_change(cls):
+        return max(cls.objects.all().values_list('last_changed', flat=True))
```

Comment:
Tohle asi funguje a je to uplne jedno (protoze jsou tam afaik nejake 3 zaznamy...) ale vhodnejsi by imho bylo pouzit Max aggregator ktery ten max executne na db serveru misto na aplikacnim serveru.

Reply:
Makes sense, zmenil jsem to